consensus/construct.go

```diff
@@ -57,7 +57,6 @@ func (consensus *Consensus) construct(
 		}
 	case msg_pb.MessageType_COMMIT:
 		if s := consensus.priKey.SignHash(payloadForSignOverride); s != nil {
-			consensusMsg.Block = consensus.block
```

Comment:
so we don't need to include the block in commit message? why there is no explanation in the commit message of about saving?

Reply:
was the block being used before? or it is totally just discarded before.